FirebaseAdmin/FirebaseAdmin/AppOptions.cs

```diff
@@ -58,5 +60,10 @@ internal AppOptions(AppOptions options)
         /// JSON.</para>
         /// </summary>
         public string ServiceAccountId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the HttpClientFactory to use when making Firebase requirests.
+        /// </summary>
+        public HttpClientFactory ClientFactory { get; set; } = new HttpClientFactory();
```

Comment:
As I understand this is C# 6 syntax. But we do target some .NET versions that are before C# 6. I'm not sure if that's a problem. The safe solution here would be to assign the default value in the constructor. WDYT?

Reply:
Making the changes now.